.clang_complete

```diff
@@ -1,2 +1,6 @@
+-std=c++1y
 -Iinc
---std=c++14
```

Comment:
c++1y was used to refer to the c++14 standard before it was completed and implemented in gcc (or whatever compiler). They are functionally equivalent now.